src/js/components/Brick.js

```diff
@@ -0,0 +1,47 @@
+// (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
+
+import React, { Component } from 'react';
+import Anchor from './Anchor';
+
+const CLASS_ROOT = "brick";
+
+export default class Brick extends Component {
+  render() {
+    var classes = [CLASS_ROOT];
+    classes.push('brick--' + this.props.width + '-' + this.props.height);
+
+    if (this.props.colorIndex) {
+      classes.push('background-color-index-' + this.props.colorIndex);
+    }
+    if (this.props.className) {
+      classes.push(this.props.className);
+    }
+
+    var label = <div className="brick--label"><span>{this.props.label}</span></div>;
+    if (this.props.route) {
+      label = <Anchor href={this.props.route} className="brick--label"><span>{this.props.label}</span></Anchor>;
+    }
+
+    return (
+      <div className={classes.join(' ')} onClick={this.props.onClick}>
+        <div className="brick--content-wrapper">
+          {this.props.children}
+        </div>
+        {label}
+      </div>
+    );
+  }
+}
+
+Brick.propTypes = {
+  label: React.PropTypes.string,
+  route: React.PropTypes.string,
+  width: React.PropTypes.number,
+  height: React.PropTypes.number,
+  colorIndex: React.PropTypes.string
+};
+
+Brick.defaultProps = {
+  width: 1,
```

Comment:
I agree that `width` and `height` props might cause some confusion. @codeswan and I have decided to replace the `width` and `height` props with just `type` prop. `type` prop can be `small`, `large`, `wide`, `tall`. https://github.com/codeswan/grommet/commit/8c8ce072b32a453a03232d96105e27c48f49e50b

I have also updated the Bricks documentation to show the information for the new `type` prop. https://github.com/codeswan/grommet-docs/commit/17356c351f4f6879a0c147b6eb11499c948c376a